pkg/agent/openflow/client.go

```diff
@@ -157,11 +151,11 @@ type Client interface {
 	// SNAT with the Openflow NAT action.
 	InstallExternalFlows(exceptCIDRs []net.IPNet) error
 
-	// InstallSNATMarkFlows installs flows for a local SNAT IP. On Linux, a
+	// InstallSNATMarkFlow installs flows for a local SNAT IP. On Linux, a
 	// single flow is added to mark the packets tunnelled from remote Nodes
 	// that should be SNAT'd with the SNAT IP. On Windows, an extra flow is
 	// added to perform SNAT for the marked packets with the SNAT IP.
-	InstallSNATMarkFlows(snatIP net.IP, mark uint32) error
+	InstallSNATMarkFlow(snatIP net.IP, mark uint32) error
```

Comment:
I feel no need to change the name in the interface. If you want to, you probably should update the comments too.

Reply:
Removed.